package.json

```diff
@@ -10,14 +10,14 @@
     "prettier": "prettier --write src/**/*.js"
   },
   "dependencies": {
-    "aframe": "https://github.com/robertlong/aframe#fix-increasing-offset",
```

Comment:
aframe.js is now a vendor asset since aframe-xr expects it to be a global. We can switch it to the official aframe CDN when the next version is released.

Reply:
I forget exactly how it failed. In any case, I was okay with making it a global script, since we wanted to take advantage of the A-Frame CDN, for caching.